frontend/packages/dev-console/integration-tests/utils/selectResourceType.ts

```diff
@@ -0,0 +1,17 @@
+import { ExpectedConditions as by, element } from 'protractor';
+
+export const resourceDeployment = element(by.id('form-radiobutton-kubernetes-field'));
+export const resourceDeploymentConfig = element(by.id('form-radiobutton-openshift-field'));
+export const resourceKnativeService = element(by.id('form-radiobutton-knative-field'));
+
+export const selectResourceDeployment = async (resourceSelected: string) => {
```

Comment:
return type `void`

Reply:
There are hundreds of utility functions in the console and we have used some of them in devconsole but never saw anyone has given the return type to a utility function.